tests/unit_tests/services/test_organise_service.py

```diff
@@ -217,3 +218,129 @@ def test_organise_project_file(self):
             mock.call(
                 os.path.join("..", "..", "..", "foo", "report-dir", "another-report-file"),
                 os.path.join(organised_project_path, "report-dir", "another-report-file"))])
+
+    def test__determine_organise_operation(self):
+        ops = ["softlink", "hardlink", "copy"]
+        self.file_system_service.symlink.return_value = ops[0]
+        self.file_system_service.hardlink.return_value = ops[1]
+        self.file_system_service.copy.return_value = ops[2]
+
+        for op in ops:
+            link_type = op[0:4]
```

Comment:
For consistency, I think we should settle for either short names (soft, hard, copy) or long names (softlink, ...) throughout the codebase.

In https://github.com/arteria-project/arteria-delivery/blob/7777dd1c37f3ffe52d1834b4597310f64e5b5061/config/organise_configs/schema.json I used long names but it doesn't really matter, you can pick the one you prefer and I'll adapt the schema if necessary.